website/docs/atmo/usage/deploying-atmo.md

```diff
@@ -52,6 +52,22 @@ To control logging in Atmo, you can use its environment variables:
 - `ATMO_LOG_LEVEL` can be set to any of `trace, debug, info, warn, error`
 - `ATMO_LOG_FILE` can be set to a file to log to \(stdout will become plaintext logs, structured logs will be written to the file\)
 
+### Tracing
+
+We have added functionality, so you can send trace data to either an OpenTelemetry collector running on the same machine, or within the same cluster, or to send data to HoneyComb directly, if you have an account with them. This only works if Atmo is running in proxy mode.
```

Comment:
If this was for a blogpost, this phrasing would be fine but for the docs I'd just keep this simply to

> When Atmo is running in [proxy mode](#link-to-proxy-mode-in-docs), it can be configured to send trace data to either ...

Reply:
Updated the wording